### ISSUE TITLE
Add tlg-catalog in `staged_dependencies.yaml` (with subdir)

### ISSUE THREAD

**Issue by Melkiades** [insightsengineering/rlistings#148]

**Comment by shajoezhu** [comment]
not a blocker for cran realeas

**Comment by shajoezhu** [comment]
hi @Melkiades and @edelarua , it seems quite outdated, do we still need to update this issue? Thanks!

**Comment by Melkiades** [comment]
@shajoezhu This is still not happening, we can also close it but notice that the divergence will not be solved if there is no routine testing of the TLG-c from staged.deps. @pawelru what do you think?

**Comment by pawelru** [comment]
We need a link from `tlg-catalog` to `rlistings` specified in `tlg-catalog` to make use of branch naming feature of stage deps. This is already there [link](https://github.com/insightsengineering/tlg-catalog/blob/dcb3099e07c5cb2a8e027d1a3b6cb5a400be75f9/package/staged_dependencies.yaml#L64-L66)
A reverse link inside `rtlistings` (i.e. `rlistings <- tlg-catalog`) would be nice for consistency but it's not actually required. So-called consistency check (that does two-way validation of deps tree) has been disabled and we are not doing this at all. 
This dependency in some form would have to be specified in the reverse-dependency-check ([issue](https://github.com/insightsengineering/idr-tasks/issues/712)) where we want to be able to check if PR in `rlistings` would break `tlg-catalog`. I don't know yet how to specify this - in particular: whether staged-deps config file will be utilised. In my opinion we shouldn't do that but it's yet to be decided in that task.
To sum up - it's nice for consistency - even if this is not executed currently.

**Comment by shajoezhu** [comment]
brilliant. let's do it then. thanks @pawelru @Melkiades

**Comment by shajoezhu** [comment]
closed in #210
